packages/api/test/e2e/doughnut.e2e.ts

```diff
@@ -0,0 +1,210 @@
+// Copyright 2019 Centrality Investments Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import testingPairs from '@polkadot/keyring/testingPairs';
+import { hexToU8a, assert } from '@polkadot/util'
+import { CENNZnut as encodeCennznut} from '@cennznet/cennznut-wasm';
+import { Doughnut as DoughnutMaker } from '@plugnet/doughnut-wasm';
```

Comment:
I suggest you create an issue for correcting the doughnut-wasm path in package.json after it is released. Then use that issue number to leave a TODO comment here such as:
// TODO #xxx fix doughnut-was path in package.json

Reply:
using the new release so this won't be required